.eslintignore

```diff
@@ -1,2 +1,3 @@
 **/*.babel.js
 coverage
+lib/shared/require-or-import-es6.js
```

Comment:
Dynamic `import()` is not yet recognized by ESLint's built-in parser. However, if you want we can lint this file using `babel-eslint`?

Reply:
Never mind. I simplified the approach, and as a side effect this is no longer an issue.